README.md

```diff
@@ -11,37 +11,29 @@ mig-controller can help you move OpenShift application workloads from a _source_
 
 ---
 
-__2. Deploy Velero to both the _source_ and _destination_ OpenShift clusters__
+__2. Use mig-operator to deploy Migration Tools to both the _source_ and _destination_ OpenShift clusters__
 
-```bash
-# Download bash script to deploy Velero along with required plugins
-wget https://raw.githubusercontent.com/fusor/mig-controller/master/hack/deploy/deploy_velero.sh
+Use mig-operator to install selected components of Migration Tooling (mig-controller, mig-ui, velero) onto your source and destination OpenShift clusters.
 
-# Login to source cluster, run 'deploy_velero.sh' against it
-oc login https://my-source-cluster:8443
-bash deploy_velero.sh
+After installing mig-operator, you can select which of mig-controller, mig-ui, and velero should be installed by creating a [MigrationController CR](https://github.com/fusor/mig-operator#migration-controller-installation):
```

Comment:
_which of mig-controller, mig-ui, and velero_ `components`

Reply:
Changed